### ISSUE TITLE
How to show historical data on NVDashborad

### ISSUE THREAD

**Issue by vilmara** [rapidsai/jupyterlab-nvdashboard#32]
Hi all, I am using the docker image `rapidsai/rapidsai:0.10-cuda10.1-runtime-ubuntu18.04`, I thought NVDashborad was installed by default; however, in order to have the "GPU DASHBOARD" menu enable I needed to run the extra installation within the docker:
```
$ pip install jupyterlab-nvdashboard
$ jupyter labextension install jupyterlab-nvdashboard
```
I can see the plots with data in realtime, how can I visualize historical data?

**Comment by jacobtomlinson** [comment]
Thanks for raising this!

We are aware of the issues with it not being bundled in the RAPIDS image, this is being resolved in rapidsai/build#43. 

Currently there is no support for historic data. This is just a real-time view.

**Comment by vilmara** [comment]
thanks!
